requirements/edx/base.txt

```diff
@@ -152,3 +152,4 @@ analytics-python==0.4.4
 
 # Needed for mailchimp(mailing djangoapp)
 mailsnake==1.6.2
+jsonfield==1.0.3
```

Comment:
Hey @jarv! I guess this line should suggest contacting `devops@edx.org` instead of listing individuals.